spec/unit/controller_spec.rb

```diff
@@ -365,5 +376,12 @@ class ControllerApplication < Flame::Application
 			inherited_controller = Class.new(ControllerController.with_actions)
 			inherited_controller.actions.should.equal ControllerController.actions
 		end
+
+		it 'should include actions from mixin module' do
+			controller_with_mixin = ControllerController
+				.class_exec { include with_actions MixIn }
+			controller_with_mixin.actions
+				.should include_actions %i[mixed another_mixed]
```

Comment:
I can add `not_include_actions`, we can't use the same method. Or i can add another argument to this method, that represents `:none?` and `:all?` methods.